src/isPositive.js

```diff
@@ -3,7 +3,7 @@ import { both, lt } from 'ramda';
 import isNumber from './isNumber';
 
 /**
- * Checks if value is a positive `Number` primitive or object.
+ * Checks if value is a positive `Number` primitive or object. Zero is not considered positive.
```

Comment:
Checks if value is a positive `Number` primitive or object. Zero is considered neither positive nor negative.